frontend/src/components/dropdowns/GenreDropdown.tsx

```diff
@@ -30,6 +31,7 @@ export default function GenreDropdown(props: GenreDropdownProps) {
       onChange={(e) => props.setSelectedGenre(e.value)}
       placeholder={"Select Genre"}
       showClear
+      disabled={props.isModifiable ?? false}
```

Comment:
I think this should default to true, not false. Probably don't want it to automatically disable on the other pages

Reply:
Done